cli/util_internal_test.go

```diff
@@ -41,6 +41,37 @@ func TestDurationDisplay(t *testing.T) {
 	}
 }
 
+func TestExtendedParseDuration(t *testing.T) {
+	t.Parallel()
+	for _, testCase := range []struct {
+		Duration   string
+		Expected   time.Duration
+		ExpectedOk bool
+	}{
+		{"1d", 24 * time.Hour, true},
+		{"1y", 365 * 24 * time.Hour, true},
+		{"10s", 10 * time.Second, true},
+		{"1m", 1 * time.Minute, true},
+		{"20h", 20 * time.Hour, true},
```

Comment:
nit: more invalid test cases? for instance 20h17h5m (hour is present 2 times)

Reply:
I tried to stick to the way Go `time.ParseDuration` works [links here](https://go.dev/play/p/c9EChF-VHkO) so it just sum all the durations even if there's multiple times the same time unit. (I discovered this behavior while working on it 🙈 )

Still, open for conversation if we want to have our own behavior.

Reply:
As long as consistent let's leave it as is 👍